version.py

```diff
@@ -30,7 +30,7 @@
 #
 # Changes should be accompanied by documentation explaining what the
 # new level means in templates/zerver/api/changelog.md.
-API_FEATURE_LEVEL = 41
+API_FEATURE_LEVEL = 42
```

Comment:
I'd use `display_settings` as the commit message prefix -- we don't use CamelCase for naming things.

Reply:
Ah! my bad.